tests/MyGame/Example/Ability.cs

```diff
@@ -26,6 +28,28 @@ public struct Ability : IFlatbufferObject
     builder.PutUint(Id);
     return new Offset<MyGame.Example.Ability>(builder.Offset);
   }
+  public AbilityT UnPack() {
+    var _o = new AbilityT();
+    this.UnPackTo(_o);
+    return _o;
+  }
+  public void UnPackTo(AbilityT _o) {
+    _o.Id = this.Id;
+    _o.Distance = this.Distance;
+  }
+  public static Offset<MyGame.Example.Ability> Pack(FlatBufferBuilder builder, AbilityT _o) {
+    if (_o == null) return default(Offset<MyGame.Example.Ability>);
+    return CreateAbility(
+      builder,
+      _o.Id,
+      _o.Distance);
+  }
+};
+
+public class AbilityT
```

Comment:
Thoughts on having the codegen emit these as separate files (perhaps as an option)? Some code linters kick up a fuss when there > 1 type per file in C#

Reply:
I think the linter is ignored by the header comment `auto-generated`, so no need to separate files.